test/parallel/test-fs-write-sync-optional-params.js

```diff
@@ -58,23 +61,25 @@ function testValid(dest, buffer, options) {
     { toString() { return 'amObject'; } },
     { [Symbol.toPrimitive]: (hint) => 'amObject' },
   ]) {
-    testInvalid(dest, 'ERR_INVALID_ARG_TYPE', badBuffer);
+    testInvalid(dest, 'ERR_INVALID_ARG_TYPE', common.mustNotMutateObjectDeep(badBuffer));
   }
 
   // First argument (buffer or string) is mandatory
   testInvalid(dest, 'ERR_INVALID_ARG_TYPE');
 
   // Various invalid options
-  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, { length: 5 });
-  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, { offset: 5 });
-  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, { length: 1, offset: 3 });
-  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, { length: -1 });
-  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, { offset: -1 });
-  testInvalid(dest, 'ERR_INVALID_ARG_TYPE', buffer, { offset: false });
-  testInvalid(dest, 'ERR_INVALID_ARG_TYPE', buffer, { offset: true });
+  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, common.mustNotMutateObjectDeep({ length: 5 }));
+  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, common.mustNotMutateObjectDeep({ offset: 5 }));
+  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, common.mustNotMutateObjectDeep({ length: 1, offset: 3 }));
+  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, common.mustNotMutateObjectDeep({ length: -1 }));
+  testInvalid(dest, 'ERR_OUT_OF_RANGE', buffer, common.mustNotMutateObjectDeep({ offset: -1 }));
+  testInvalid(dest, 'ERR_INVALID_ARG_TYPE', buffer, common.mustNotMutateObjectDeep({ offset: false }));
+  testInvalid(dest, 'ERR_INVALID_ARG_TYPE', buffer, common.mustNotMutateObjectDeep({ offset: true }));
```

Comment:
In this file, as an example, it looks at least the fourth argument to `testInvalid` is always passed to `common.mustNotMutateObjectDeep`. Can we move such occurrences into `testInvalid` etc. to reduce bloat?